benchmarks/graphics_pipeline/main.cpp

```diff
@@ -65,6 +66,8 @@ static constexpr std::array<const char*, kAvailableScenes.size()> kAvailableScen
 
 static constexpr uint32_t kPipelineCount = kAvailablePsShaders.size() * kAvailableVsShaders.size();
 
+static constexpr uint32_t kMaterialCount = 5;
```

Comment:
Add a comment to explain this limit is arbitrary as for now, we don't need more than 5 material IDs.
(Also nit: rename to `kMaxMaterialCount` to explicitly say this is a limit, not the actual count).

Reply:
Done.